app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
They have to tab through everything to focus on it, so they can hear what it is before they use it. However, we can create access keys to get people directly to the submit button rather than tabbing through the whole thing. We also need to inform people ahead of time that we have some controls mapped to certain things

http://webaim.org/techniques/keyboard/accesskey#spec

So if we bind enter to the submit button, we should instruct folks at the top of the form to use the spacebar for the add/remove buttons, a defined access key, and enter to submit the form.